File: 961-n-repeated-element-in-size-2n-array/961-n-repeated-element-in-size-2n-array.c
int repeatedNTimes(int* A, int ASize) {
    int hash[10000] = {0};
    
    for (int i = 0; i < ASize; i++) {
        hash[A[i]]++;
        if (hash[A[i]] > 1)
            return A[i];
    }
    return 0;
}

File: 961-n-repeated-element-in-size-2n-array/961-n-repeated-element-in-size-2n-array.java
class Solution {
    public int repeatedNTimes(int[] nums) {
        int[] arr = new int[10001];
        for(int i = 0; i < nums.length; i++){
            arr[nums[i]]++;
            if(arr[nums[i]] == 2)
                return nums[i];
        }
        return nums[0];
    }
}